packages/web/src/components/table-handle/table-handle-dnd-indicator/types.ts

```diff
@@ -0,0 +1,18 @@
+import type {
+  EventDeclarations,
+  PropDeclarations,
+} from '@aria-ui/core'
+import type { Editor } from '@prosekit/core'
+import type { TableCommandsExtension } from '@prosekit/extensions/table'
+
+export interface TableHandleDndIndicatorProps {
```

Comment:
I personally prefer not to use abbreviations in the custom element names. Do you think it would be a good idea to rename `Dnd` to `Drag`?

Reply:
Done https://github.com/prosekit/prosekit/pull/1109/commits/7344e41a7bb6c65ec046535f696b830a692c58b8